setup/www/resources/config/nodejs.org

```diff
@@ -301,6 +301,16 @@ server {
         default_type text/plain;
     }
 
+    location /security.txt {
+        alias /home/www/nodejs/static/security.txt;
```

Comment:
Per [spec draft](https://tools.ietf.org/html/draft-foudil-securitytxt-03#section-4), this should be a redirect to `/.well-known/security.txt`.

Reply:
More specifically:

> A security.txt file located under the top-level path SHOULD either redirect to the security.txt file under the /.well-known/ path or be used as a fall back.

I guess the current implementation above can be considered a fallback, but IANAL :).

I don't have strong feelings about this, happy to configure a redirect.

Reply:
The scheme just above that section suggests a redirect. I would prefer this to be a redirect — that would hint that this is a part of some spec, not just a random file.